setup.py

```diff
@@ -41,7 +41,7 @@ def main():
             "s3": ["boto3"],
             "salesforce": ["simple-salesforce"],
             "sftp": ["paramiko"],
-            "slack": ["slackclient"],
+            "slack": ["slackclient==1.3.0"],
```

Comment:
Thanks @crayolakat! `setup.py` files should be as loose with versions as possible (to increase compatibility), so does this look right? Should work with anything >= 1 but less than 2
```suggestion
            "slack": ["slackclient~=1"],
```

Reply:
Yes, thank you! I committed the suggestion.